docs/keyless-signing.md

```
@@ -0,0 +1,138 @@
+
+```bash
+caib image build manifest.aib.yml \
+  --name my-build \
+  --arch aarch64 \
+  --push quay.io/<user>/my-build:bootc \
+  --push-disk quay.io/<user>/my-build:disk
+```
+
+On completion, the output shows the image digest that Chains will sign:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)